### PR TITLE
docs(review): updates link and formatting from doc review

### PR DESCRIPTION
**Pull request by PaulRMellor**
**Documentation**

Introduces a couple of small doc changes to fix an internal link and make some formatting/content more consistent

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

File: documentation/modules/configuring/proc-manual-delete-pod-pvc-kafka.adoc [hunk 1]
```diff
@@ -20,13 +20,12 @@ The following procedure should only be performed if you have encountered storage
 
 . Find the name of the `Pod` that you want to delete.
 +
-Kafka broker pods are named _<cluster-name>_-kafka-_<index>_, where _<index>_ starts at zero and ends at the total number of replicas minus one.
+Kafka broker pods are named <cluster_name>-kafka-<index_number>, where <index_number> starts at zero and ends at the total number of replicas minus one.
```
Comment (scholzj) [inline documentation/modules/configuring/proc-manual-delete-pod-pvc-kafka.adoc:23]:
```suggestion
Kafka broker pods are named `<cluster_name>-kafka-<index_number>`, where `<index_number>` starts at zero and ends at the total number of replicas minus one.
```

Would it make sense to format it as code to make it more readable?

File: documentation/modules/configuring/proc-manual-delete-pod-pvc-zookeeper.adoc [hunk 2]
```diff
@@ -20,13 +20,12 @@ The following procedure should only be performed if you have encountered storage
 
 . Find the name of the `Pod` that you want to delete.
 +
-ZooKeeper pods are named _<cluster-name>_-zookeeper-_<index>_, where _<index>_ starts at zero and ends at the total number of replicas minus one.
+ZooKeeper pods are named <cluster_name>-zookeeper-<index_number>, where <index_number> starts at zero and ends at the total number of replicas minus one.
```
Comment (scholzj) [inline documentation/modules/configuring/proc-manual-delete-pod-pvc-zookeeper.adoc:23]:
```suggestion
ZooKeeper pods are named `<cluster_name>-zookeeper-<index_number>`, where `<index_number>` starts at zero and ends at the total number of replicas minus one.
```

File: documentation/modules/managing/proc-manual-rolling-update-pods.adoc [hunk 3]
```diff
@@ -43,14 +43,18 @@ spec:
...
-The _index_ is a number assigned to the `Pod`.
-
-. Annotate the `Pod` resource in Kubernetes.
+Pod naming conventions are as follows:
++
+--
+* <cluster_name>-kafka-<index_number> for a Kafka cluster
+* <cluster_name>-zookeeper-<index_number> for a ZooKeeper cluster
+* <cluster_name>-connect-<index_number> for a Kafka Connect cluster
+* <cluster_name>-mirrormaker2-<index_number> for a MirrorMaker 2 cluster
```
Comment (scholzj) [inline documentation/modules/managing/proc-manual-rolling-update-pods.adoc:52]: Same as above, should the names be formatted as code?

File: documentation/modules/managing/proc-manual-rolling-update-pods.adoc [hunk 4]
```diff
@@ -43,14 +43,18 @@ spec:
...
+--
 +
-Use `kubectl annotate`:
+The <index_number> assigned to a pod starts at zero and ends at the total number of replicas minus one.
```
Comment (scholzj) [inline documentation/modules/managing/proc-manual-rolling-update-pods.adoc:55]:
```suggestion
The `<index_number>` assigned to a pod starts at zero and ends at the total number of replicas minus one.
```

**Review by scholzj — APPROVED** [review]
Thanks.

**Review by ppatierno — APPROVED** [review]
LGTM

**Pull request merged**